Pages/Checkout/checkout.css

```diff
@@ -0,0 +1,18 @@
+.doc-success{
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
Suggestion:
You can declare these properties in util and use it as class, as it can be used in multiple areas.

Reply:
Noted, thanks!